ykbh/src/lib.rs

```diff
@@ -15,8 +15,27 @@ pub struct SIRInterpreter {
     layout: Layout,
 }
 
+impl StackFrame {
+    fn free(&mut self) {
+        unsafe { dealloc(self.locals, self.layout) }
+    }
+}
+
+pub struct SIRInterpreter {
+    frames: Vec<StackFrame>,
+    bbidx: usize,
```

Comment:
Could that be `ykpack::BasicBlockIndex`? Maybe an underscore too? `bb_idx`, or just `bb` even?

Reply:
Fixed in 0d5cec2.